src/link/filelink.cpp

```diff
@@ -104,9 +104,7 @@ bool FileLink::isOpen()
 bool FileLink::finishConnection()
 {
     // Only close files that are open
-    if(_file.isOpen()) {
-        _file.close();
-    }
+    _file.close();
```

Comment:
IIRC this will result in a warning message saying that it's closing something that is not open.